.claude/skills/error-handling/reference/examples/error-patterns.py

```
@@ -0,0 +1,801 @@
+    user = request.user
+
+    # Check subscription status
+    if not hasattr(user, 'subscription') or not user.subscription.is_active:
+        raise PaymentRequired({
+            'detail': 'Active subscription required',
+            'subscription_plans': '/api/subscriptions/plans/',
+            'trial_available': True,
+        })
+
+    # Check business rules
+    daily_posts = user.posts.filter(created_at__date=timezone.now().date()).count()
```

Ruff (0.14.10)

Undefined name `timezone`

(F821)